wwwapp/urls.py

```diff
@@ -46,6 +46,7 @@
     url(r'^template_for_workshop_page/$', views.template_for_workshop_page_view, name='template_for_workshop_page'),
     url(r'^program/$', RedirectView.as_view(url='/%d/program/' % CURRENT_YEAR, permanent=False), name='program'),
     url(r'^([0-9]+)/program/$', views.program_view, name='year_program'),
+    url(r'^resource_auth/$', views.resource_auth_view),
```

Comment:
Add name

Reply:
Name is for referencing in other parts of code. This does not need to be referenced.

Reply:
But every single other endpoint has a name, including dataForPlan which doesn't need to be referenced anywhere either.